src/ui.data-table/UTable.vue

```diff
@@ -633,9 +627,10 @@ function setHeaderCellWidth() {
   const mainHeaderItems = [...(headerRowRef.value?.children || [])];
   const stickyHeaderItems = [...(stickyHeaderRowRef.value?.children || [])];
 
-  stickyHeaderItems.forEach((item, index) => {
-    item.style.width = `${mainHeaderItems[index]?.offsetWidth / PX_IN_REM}rem`;
-  });
+  stickyHeaderItems.map((item, index) => ({
```

Comment:
map retuns a new array. So where is you use it?

Reply:
Returned as it was before. We manipulate with DOM here, so we don't need any return value.